docs/source/handlers.rst

```diff
@@ -34,6 +34,37 @@ Complete list of generic handlers
     state_param_scheduler.StateParamScheduler
 
 
+Checkpoint Events
+-----------------
+
+.. versionadded:: 0.5.0
```

Comment:
This should be added to the docstrings. Can you locally render the docs and show how this look like?
```suggestion
.. versionadded:: 0.5.3
```